railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -121,7 +121,7 @@ def config_when_updating
       action_cable_config_exist = File.exist?("config/cable.yml")
       rack_cors_config_exist = File.exist?("config/initializers/cors.rb")
       assets_config_exist = File.exist?("config/initializers/assets.rb")
-      new_framework_defaults_5_1_exist = File.exist?("config/initializers/new_framework_defaults_5_1.rb")
+      new_framework_defaults_5_2_exist = File.exist?("config/initializers/new_framework_defaults_5_2.rb")
```

Comment:
`new_framework_defaults_5_2_exist` is now an unused variable. It is getting reported as a warning from the test:

```
rails/railties/lib/rails/generators/rails/app/app_generator.rb:124: warning: assigned but unused variable - new_framewo
rk_defaults_5_2_exist
```